string type
float64 size
float64 resolution
---
string mesh_path